cli/src/pcluster/models/imagebuilder_extra_attributes.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+from pcluster.models.imagebuilder import ImagebuilderDevSettings
+
+
+class ChefAttributes:
+    """Attributes for Chef Client."""
+
+    def __init__(self, dev_settings: ImagebuilderDevSettings):
+        self.cfn_region = None
+        self.nvidia = None
+        self.is_official_ami_build = None
+        self.custom_node_package = None
+        self.cfn_base_os = None
+        self._extra_attributes_json = {}
+        self._set_default(dev_settings)
+        self._set_extra_attributes(dev_settings)
+
+    def _set_default(self, dev_settings: ImagebuilderDevSettings):
+        self.cfn_region = "${AWS::Region}"
+        self.nvidia = {"enabled": "false"}
+        self.is_official_ami_build = str.lower(str(dev_settings.update_os_and_reboot)) if dev_settings else "false"
```

Comment:
lets check presence of `dev_settings.update_os_and_reboot` not just `dev_settings`

Reply:
Done